include/dali/core/util.h

```diff
@@ -182,6 +182,16 @@ struct same_as { using type = T; };
 template <typename T>
 using same_as_t = typename same_as<T>::type;
 
+
+struct identity {
+  template <typename T>
+  DALI_HOST_DEV DALI_FORCEINLINE
+  T &&operator()(T &&x) const noexcept {
+    return std::forward<T>(x);
+  }
+};
```

Comment:
Could you document it? It's hard for me to understand from the implementation, what the `identity` is for

Reply:
It's for... doing nothing to the argument :)

Reply:
https://en.cppreference.com/w/cpp/utility/functional/identity